File: encode_images.js
const fs = require('fs');
const path = require('path');

const dir = path.join(__dirname);
const logoData = fs.readFileSync(path.join(dir, 'LOgoQR.jpg'));
const qrloaData = fs.readFileSync(path.join(dir, 'QRLOA.png'));

const logoB64 = logoData.toString('base64');
const qrloaB64 = qrloaData.toString('base64');

console.log('LOGO_LEN=' + logoB64.length);
console.log('QRLOA_LEN=' + qrloaB64.length);

fs.writeFileSync(path.join(dir, 'logo_b64.txt'), logoB64, 'utf8');
fs.writeFileSync(path.join(dir, 'qrloa_b64.txt'), qrloaB64, 'utf8');
console.log('Done');

File: encode_images.py
import base64, os
d = os.path.dirname(os.path.abspath(__file__))
with open(os.path.join(d,'LOgoQR.jpg'),'rb') as f:
    logo = base64.b64encode(f.read()).decode()
with open(os.path.join(d,'QRLOA.png'),'rb') as f:
    qrloa = base64.b64encode(f.read()).decode()
print(f"LOGO_LEN={len(logo)}")
print(f"QRLOA_LEN={len(qrloa)}")
with open(os.path.join(d,'logo_b64.txt'),'w') as f: f.write(logo)
with open(os.path.join(d,'qrloa_b64.txt'),'w') as f: f.write(qrloa)
print("Done")
